tests/scattering1d/test_numpy_scattering1d.py

```diff
@@ -27,9 +28,93 @@ def test_Scattering1D(self, backend):
         Q = data['Q']
         Sx0 = data['Sx']
 
-        T = x.shape[-1]
+        N = x.shape[-1]
 
-        scattering = Scattering1D(J, T, Q, backend=backend, frontend='numpy')
+        scattering = Scattering1D(J, N, Q, backend=backend, frontend='numpy')
 
         Sx = scattering(x)
         assert np.allclose(Sx, Sx0)
+    
+    def test_Scattering1D_T(self, backend):
+        """
+        Applies scattering on a stored signal to make sure its output agrees with
+        a previously calculated version.
+        """
+        test_data_dir = os.path.dirname(__file__)
+
+        with open(os.path.join(test_data_dir, 'test_data_1d.npz'), 'rb') as f:
+            buffer = io.BytesIO(f.read())
+            data = np.load(buffer)
+
+        x = data['x']
+        J = data['J']
+        Q = data['Q']
+        Sx0 = data['Sx']
+
+        N = x.shape[-1]
+
+        # default
+        scattering1 = Scattering1D(J, N, Q, backend=backend, frontend='numpy')
+        Sx1 = scattering1(x)
+ 
+        # adjust T
+        sigma_low_scale_factor = 2
+        T=2**(J-sigma_low_scale_factor)
+        scattering2 = Scattering1D(J, N, Q, T=T, backend=backend, frontend='numpy')
+        Sx2 = scattering2(x)
+        
+        # for now, just be sure that the output shape is different from default
+        # should we add this new result to the test data?
+        #print('Sx1 shape: ' + str(Sx1.shape) + ' Sx2.shape: ' + str(Sx2.shape))
+        assert Sx2.shape == (Sx1.shape[0], Sx1.shape[1], Sx1.shape[2]*2**(sigma_low_scale_factor))
+    
+    def Scattering1D_filter_factory_T_plot(self, N, J, T, phi_f, psi1_f, msg):
```

Comment:
Can go into an example?

Reply:
ok sure, examples/1d is a better place for the plots

Reply:
done